security/fma-holocene.md

```diff
@@ -0,0 +1,196 @@
+
+We are performing multi-client testing (op-geth and op-reth) and will run a multi-client devnet, so bugs in either will be quickly detected by consensus disagreements among them.
+
+
+## Update to the MIPS contract
+
+As part of the Holocene change, the Go compiler was updated from 1.21 to 1.22. This impacts the op-program as the go1.22 runtime makes additional syscalls that are not supported by the pre-Holocene MIPS Fault Proof Virtual Machine (FPVM).
+As such, Holocene includes an update to MIPS.sol that supports go1.22 programs. This change to the FPVM is very minimal; an update to the `fcntl` syscall emulation that was partially implemented by the pre-Holocene FPVM.
+
+### Undetected go1.22 changes
+
+If there were other changes introduced by go1.22 beyond `fcntl`, and the MIPS FPVM does not emulate them correctly, it could result in blocks that are not fault-provable. This ultimately results in fault dispute games resolving incorrectly.
+
+#### Risk Assessment
+
+medium severity / low likelihood
+
+#### Mitigations
+
+[An audit](https://gist.github.com/3docSec/068537844f2ddb204324079138f14551) of the go1.22 changes on the op-program and the FPVM was performed by [3DOC Security](https://x.com/3docSec). The audit did not find any problems related to the go1.22 changes that breaks op-program execution in the FPVM.
+
+Furthermore, any game that resolves incorrectly is subject to the 3.5-day finality delay. This gives the Security Council ample time to detect and respond to invalid games (including blacklisting games and falling back to to permissioned games).
+
+Action item: Update op-sepolia vm-runner to use the new FPVM. The vm-runner runs the op-program in the MIPS FPVM using inputs sampled from a live chain. Having the vm-runner run the op-program on op-sepolia for a couple days will increase confidence that the network will continue to be fault provable.
+
+#### Detection
+
+`dispute-mon` detects invalid games and forecasts those that will be resolved incorrectly.
+
+### Divergent FPVM implementations
+
+If the offchain FPVM behaves differently from the MIPS FPVM contract, the op-challenger will be unable to act honestly.
+
+#### Risk Assessment
+
+medium severity / low likelihood
+
+#### Mitigations
+
+The offchain FPVM, unencumbered by governance, can be updated at any time to match the behavior of the MIPS contract.
+
+#### Detection
+
+To reduce the risk of discrepancies, we differentially test the MIPS contract against the offchain FPVM implementation.
+
+## Generic Items
+
+See [./fma-generic-hardfork.md](./fma-generic-hardfork.md).
+
+- [ ] Check this box to confirm that these items have been considered and updated if necessary.
+
+## Audit Requirements
+
+This may require a mini audit due to updates to the MIPS contract.
+
+## Action Items
```

Comment:
A comment I could leave on all of the Mitigations sections, but will just leave here is that the mitigations listed seem good, but it's unclear if there is an actual plan and intent to do them. 

I'd suggest summarizing and prioritizing them here. Some should be deemed blocking and others nice to have. They can then be ported to the github project (if not already tracked there).

Reply:
[a969589](https://github.com/ethereum-optimism/design-docs/pull/127/commits/a9695896ff32b3019801e7af59fe46b0ae59f5f8)

Reply:
LGTM